p2p/types/mocks/mocks.go

```diff
@@ -0,0 +1 @@
+package mock_types
```

Comment:
Optional: we can add a comment that says: `This file is in place to declare the package for dynamically generated mocks.`

Reply:
Makes sense, added also for shared mocks

https://github.com/pokt-network/pocket/pull/382/commits/a7b1d96c7c01e974eeb6ad054def04b0fee2c73b